clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/model/AmazonServerGroup.groovy

```diff
@@ -37,6 +38,15 @@ class AmazonServerGroup implements ServerGroup, Serializable {
   Map<String, Object> image
   Map<String, Object> launchConfig
   Map<String, Object> launchTemplate
+
+  /**
+   * The fields marked @Alpha are subject to change in the future due to addition of newer ASG features like multiple launch templates.
+   */
+  @Alpha
+  Map<String, Object> mixedInstancesPolicy
+  @Alpha
+  Map<String, Object> launchTemplateSpecification
```

Comment:
There will probably be more launch settings that will need to be extracted out of either `launchTemplate` or `mixedInstancesPolicy` going forward. Hence the `Alpha` annotation.